code/addons/a11y/src/components/A11YPanel.tsx

```diff
@@ -4,16 +4,16 @@ import { styled } from '@storybook/theming';
 
 import { ActionBar, Icons, ScrollArea } from '@storybook/components';
 
-import { AxeResults } from 'axe-core';
+import type { AxeResults } from 'axe-core';
 import { useChannel, useParameter, useStorybookState } from '@storybook/api';
-// eslint-disable-next-line import/no-cycle
+
 import { Report } from './Report';
-// eslint-disable-next-line import/no-cycle
```

Comment:
I'm not sure why these comments were removed when running eslint with `--fix`, but only three components began erroring, and I added the ignore statements back for those.